src/Analyzers/CSharp/CodeFixes/CSharpCodeFixes.projitems

```diff
@@ -19,6 +19,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)ConvertTypeOfToNameOf\CSharpConvertTypeOfToNameOfCodeFixProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)ConvertSwitchStatementToExpression\ConvertSwitchStatementToExpressionCodeFixProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)ConvertSwitchStatementToExpression\ConvertSwitchStatementToExpressionCodeFixProvider.Rewriter.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)CSharpUsePatternCombinatorsCodeFixProvider.cs" />
```

Comment:
I am wondering if we should add `ExportCodeFixProviderAttribute` and `DiagnosticAnalyzerAttribute` to Banned APIs in Features layer, with a recommendation to move the analyzer/fixer down to the shared layer, when possible. For cases where this is not possible due to analyzer/fixer using a public API not available in a shipped Roslyn version, one would be expected to add a source suppression, with a TODO tracking the move later when the public API does become available (similar to the issue closed by this PR).

What do you thing @Youssef1313?

Reply:
Btw, I don't recall if BannedApiAnalyzer flags banned attribute constructors :-)

Reply:
@mavasani There are too many codefixes there anyway. Banning the attribute (if possible) and suppressing all of the warnings is going to be noisy. I may look into moving more codefixes later. Then fix BannedApiAnalyzer if needed and use it.